### PR TITLE
Improve show acl commands

### PR DESCRIPTION
**Pull request by bingwang-ms**
#### What I did
HLD https://github.com/sonic-net/SONiC/pull/1261
This PR is to enhance `show acl table` and `show acl rule` commands.
Currently, `show acl table` and `show acl rule` commands read ACL table/rule configuration from `CONFIG_DB` directly. We don't know whether the ACL table or rule is created successfully.
We improved `swss` to write the status of ACL table/rule into a `STATE_DB` table. In this PR, the show command is enhanced to read the status from `STATE_DB` table.
#### How I did it
1. Introduce two tables in `STATE_DB`
2. `orchgent` writes the status to `STATE_DB`
3. show commands read the status from `STATE_DB`.

#### How to verify it
Verified by copying the new script to a testbed, and check the output.

#### Previous command output (if the output of a command-line utility has changed)
```
$ show acl table DATAACL
Name     Type    Binding      Description    Stage     
-------  ------  -----------  -------------  -------   
DATAACL  L3      Ethernet0    DATAACL        ingress   
                 Ethernet4
                 Ethernet8
                 Ethernet12
```
```
show acl rule
Table    Rule          Priority    Action    Match               
-------  ------------  ----------  --------  ------------------- 
DATAACL  RULE_1        9999        DROP      DST_IP: 9.5.9.3/32  
                                             ETHER_TYPE: 2048
DATAACL  RULE_2        9998        FORWARD   DST_IP: 10.2.1.2/32 
                                             ETHER_TYPE: 2048
                                             IP_PROTOCOL: 6
                                             L4_DST_PORT: 22
```
#### New command output (if the output of a command-line utility has changed)
```
$ show acl table DATAACL
Name     Type    Binding      Description    Stage      Status
-------  ------  -----------  -------------  -------    -------
DATAACL  L3      Ethernet0    DATAACL        ingress    Active
                 Ethernet4
                 Ethernet8
                 Ethernet12
```
```
show acl rule
Table    Rule          Priority    Action    Match                Status
-------  ------------  ----------  --------  -------------------  --------
DATAACL  RULE_1        9999        DROP      DST_IP: 9.5.9.3/32   Active
                                             ETHER_TYPE: 2048
DATAACL  RULE_2        9998        FORWARD   DST_IP: 10.2.1.2/32  Active
                                             ETHER_TYPE: 2048
                                             IP_PROTOCOL: 6
                                             L4_DST_PORT: 22
```

### CODE REVIEW

**Review by ZhaohuiS — DISMISSED** [review]
LGTM

**Comment by prsunny** [comment]
How about the ACL rules not handled by orchagent?

**Comment by bingwang-ms** [comment]
> How about the ACL rules not handled by orchagent?

You mean the ACL rules added by pfcorch, muxorch or SDK? For such ACL rules, since there is no CONFIG_DB entry, so the show command can't show them.
Since such rules are not added by end user, I think it also makes sense to not show them in the show command.

File: acl_loader/main.py [hunk 1]
```diff
@@ -786,32 +826,36 @@ def show_table(self, table_name):
...
         """
-        header = ("Name", "Type", "Binding", "Description", "Stage")
+        header = ("Name", "Type", "Binding", "Description", "Stage", "Status")
 
         data = []
         for key, val in self.get_tables_db_info().items():
             if table_name and key != table_name:
                 continue
-
+
```
Comment (prsunny) [inline acl_loader/main.py:835]: Extra space?

**Review by prsunny — APPROVED** [review]
lgtm

**Comment by StormLiangMS** [comment]
@bingwang-ms could you help to raise PR for 202211?

**Comment by bingwang-ms** [comment]
> @bingwang-ms could you help to raise PR for 202211?

PR raised.

**Pull request merged**